src/controller/python/chip/clusters/Attribute.py

```diff
@@ -199,21 +205,24 @@ def _handleAttributeData(self, path: AttributePathWithListIndex, status: int, da
             attributeType = _AttributeIndex.get(str(AttributePath(
                 ClusterId=path.ClusterId, AttributeId=path.AttributeId)), None)
             attributeValue = None
+            tlvData = chip.tlv.TLVReader(data).get().get("Any", {})
             if attributeType is None:
-                attributeValue = chip.tlv.TLVReader(data).get().get("Any", {})
+                attributeValue = ValueDecodeFailure(
+                    tlvData, LookupError("attribute schema not found"))
             else:
                 try:
-                    attributeValue = attributeType.FromTLV(data)
-                except:
+                    attributeValue = attributeType(attributeType.FromTLV(data))
+                except Exception as ex:
                     logging.error(
                         f"Error convering TLV to Cluster Object for path: Endpoint = {path.EndpointId}/Cluster = {path.ClusterId}/Attribute = {path.AttributeId}")
                     logging.error(
                         f"Failed Cluster Object: {str(attributeType)}")
-                    raise
+                    attributeValue = ValueDecodeFailure(
```

Comment:
Also, I don't think it's necessary to pass the raw TLV through - that's just noisy.

We already print out a lot of valuable information about what went wrong in the lines above, that should be sufficient.